app/src/components/market/market_home.tsx

```diff
@@ -38,11 +42,16 @@ export const MarketHome: React.FC<Props> = (props: Props) => {
         />
       )}
       <ListCard>
-        {RemoteData.is.success(markets)
-          ? markets.data.map((item, index) => {
-              return <ListItem key={index} data={item}></ListItem>
+        {RemoteData.hasData(markets)
+          ? markets.data.slice(0, props.count).map(item => {
```

Comment:
I think we can take advantage of the destructuring in the line 31, and add `count` and `onShowMore` to the destructuring

Reply:
Just for the record, I kind of like to use `props.foo` directoy when `foo` is only used once. That being said, I changed it for consistency with what we're already doing.